python_modules/dagster/dagster/_core/definitions/load_asset_checks_from_modules.py

```diff
@@ -0,0 +1,70 @@
+from types import ModuleType
+from typing import Generator, Iterable, Optional, Sequence
+
+import dagster._check as check
+
+from .asset_checks import AssetChecksDefinition
+from .events import (
+    CoercibleToAssetKeyPrefix,
+    check_opt_coercible_to_asset_key_prefix_param,
+)
+
+
+def _find_checks_in_module(
```

Comment:
Should we make something like a  `_find_objects_in_module_of_type` function to avoid redundant implementation with `_find_assets_in_module`?